airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/AbstractJdbcCompatibleSourceOperations.java

```diff
@@ -217,7 +217,7 @@ protected void setInteger(final PreparedStatement preparedStatement, final int p
   }
 
   protected void setBigInteger(final PreparedStatement preparedStatement, final int parameterIndex, final String value) throws SQLException {
-    preparedStatement.setLong(parameterIndex, Long.parseLong(value));
+    preparedStatement.setLong(parameterIndex, new BigDecimal(value).toBigInteger().longValue());
```

Comment:
Since this change is in `AbstractJdbcCompatibleSourceOperations.java`, will it be safe to assume that its correct for all the sources that use this class?

Reply:
That's the assumption - that by sending it through BigDecimal it will be safer in case of incoming string in notation other than plain integer